mod-di-converter-storage-server/src/test/java/org/folio/rest/impl/MappingProfileTest.java

```diff
@@ -725,7 +725,6 @@ public void shouldUpdateProfileOnPutAndReplaceExistingAssociationWithActionProfi
       .post(MAPPING_PROFILES_PATH)
       .then()
       .statusCode(HttpStatus.SC_CREATED)
-      .body("profile.name", is("Boo"))
```

Comment:
The following check has been removed, because after all mappingProfile2' profile name will be updated to profile name => "mapping profile 2".